README.md

```diff
@@ -158,17 +158,41 @@ end
 ```
 
 #### Email Verification
+
+This example shows how to request verification of a single email address. 
+
 ```ruby
 begin
-  result = AddressFinder.email_verification(email: 'john.doe')
+  result = AddressFinder.email_verification(email: 'john.doe', features: "domain,connection")
   $stdout.puts "This email address is verified: #{result.is_verified}"
 rescue AddressFinder::RequestRejectedError => e
   response = JSON.parse(e.body)
   $stdout.puts response['message']
 end
 ```
 
+This example shows how to request verification of several email addresses:
+
+```ruby 
+emails = [
+  "bert@myemail.com", 
+  "charlish@myemail.com", 
+  "support@addressfinder.com", 
+  "bad-email-address"
+]
+
+results = AddressFinder.email_verification_batch(emails: emails, concurrency: 2, features: "provider,domain,connection")
```

Comment:
Just wondering the benefit of adding `provider` here?  Would that not merit it's own separate "how to"?

Reply:
I think the _gem_ could really do with some improved instructions in general. I will make an improved howto guide for the batch verification when I do the release.